pennylane/debugging.py

```diff
@@ -105,3 +110,198 @@ def get_snapshots(*args, **kwargs):
         return dbg.snapshots
 
     return get_snapshots
+
+
+class PLDB(pdb.Pdb):
+    """Custom debugging class integrated with Pdb.
```

Comment:
Might be nice to link "Pdb" with the python debugger.

Reply:
This will be addressed in the docs PR 👍🏼